pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Describe your changes
+
+## Issue ticket number and link
+
+Fixes # (issue)
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code
```

Comment:
I'm good with this checklist if it's helpful to people and stays simple 👍 I just have a personal gripe with PR checklists because it got unnecessarily large at my last job and clearly slowed down the team. 

My opinion is that the first point is implicit to any PR, and the rest should preferably be automated via ci (I believe we're close to automating the last point? As that would save significant time for small changes)

Reply:
Thank you for your feedback! Sure, we can delete the first point if you think it’s redundant. I think we could remove the others once this will be automated. If any of the checkpoints are not required for the PR, they can simply be deleted.